Readability.js

```diff
@@ -724,8 +734,12 @@ Readability.prototype = {
         // Add the score to the parent. The grandparent gets half.
         parentNode.readability.contentScore += contentScore;
 
-        if (grandParentNode)
+        if (grandParentNode && grandParentNode.readability) {
           grandParentNode.readability.contentScore += contentScore / 2;
+          if (greatGrandParentNode && greatGrandParentNode.readability) {
+            greatGrandParentNode.readability.contentScore += contentScore / 4;
```

Comment:
Mm... I've wondered before now if we should just use some way of scoring that deals with this automagically (for whatever depth) but I've not figure out how best to do this. If this works, let's start here, I guess.